.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature
-about: Document a piece of functionality or feature that needs to be developed.
+about: Track that something new needs to be created
```

Comment:
Or just simply "Something new that needs to be created"

Reply:
Done